lib/commands/list-meetings.js

```diff
@@ -0,0 +1,37 @@
+const BBB = require("../BBB");
+const { getLogger } = require("../command-helpers");
+const _ = require("lodash/fp");
+
+module.exports = {
+  name: "list-meetings",
+
+  description: "List meetings currently running on BBB",
+
+  options: (yargs) => yargs,
```

Comment:
is it mandatory ? Because it looks like useless

Reply:
Right, I should make it optional and handle that in `cli.js`.